doc/extdev/parserapi.rst

```diff
@@ -3,6 +3,35 @@
 Parser API
 ==========
 
+`The document of docutils describes`__ about the parsers like following:
+
+    The Parser analyzes the the input document and creates a node tree
+    representation.
+
+__ http://docutils.sourceforge.net/docs/dev/hacking.html#parsing-the-document
+
+In Sphinx, the parser modules works as same as docutils.  The parsers are
+registered to Sphinx by extensions using Application APIs;
+:meth:`Sphinx.add_source_suffix()` and :meth:`Sphinx.add_source_parsers()`.
+
+The *source suffix* is a mapping from file suffix to file type.  For example,
+``.rst`` file is mapped to ``'restructuredtext'`` type.  Sphinx uses the
+file type to looking for parsers from registered list.  On searching,
+Sphinx refers ``Parser.supported`` attribute, and picks up a parser which
```

Comment:
refers to the ``Parser.supported`` attribute and picks

(missing 'to the'. Drop the comma before 'and')

Reply:
done